Proxy/LB update for templates with Azure Linux

guardrex opened the pull request:
Addresses https://github.com/aspnet/AspNetCore/issues/4135

Set for 2.1 and 2.2. Opened https://github.com/aspnet/AspNetCore.Docs/issues/12737 to revert the "Until new container images are provided in Azure" remark at 3.0 RTM.

bradygaster commented:
dude, @guardrex is 🥇 🥇 🥇 💯 

@Tratcher i think this should go into the "3.0 preview" moniker before we publish the blog post. If we get this link out there, I will aka it, then set the aka to the preview link. That way customers can bounce right to it from the blog post. And, frankly, I think @guardrex's content is nearly copypastable into a blog post. 

Lunch/beer/coffee on me the next time we're in the same room!

guardrex commented:
All credit to @Tratcher on this one. I just **_stole his text_** and sprinkled in a few commas and whatnot.

tdykstra commented on aspnetcore/host-and-deploy/proxy-load-balancer.md:
```diff
@@ -305,6 +305,38 @@ services.Configure<ForwardedHeadersOptions>(options =>
 
+::: moniker range=">= aspnetcore-2.1 <= aspnetcore-2.2"
+
+## .NET Core templates fail in Azure Linux due to missing scheme forwarders
```
Could simplify this to Add scheme forwarders for Linux.

Also, this might fit better before Troubleshooting rather than after it, so Troubleshooting remains the last section before Next steps. Or make this an H3 under Troubleshooting and change the heading text to indicate the error text you'd see before adding Forwarded Headers Middleware.

guardrex commented:
@tdykstra Take one more quick :eyes: ... I ended up with a different section heading. I moved the section *UP*. I adjusted the length of the code lines in the example.
